.gitignore

```diff
@@ -155,3 +155,9 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
+# .idea and models
+.idea/
+classifiers/*.model
+classifiers/*.model.*
+classifiers/*.pkl
+data/*.csv
```

Comment:
What if we want to push the datasets?

Reply:
We may un-comment data/*.csv to include data-set, concerning classifiers they are to be generated when running the code locally. 
However, we may un-comment classifiers as well in case models are needed to persist on the repository for clone then just load the available models for prediction. what do you think?

Reply:
I think that's a good idea, let's un-comment classifiers too.